detection.py

```diff
@@ -153,3 +155,29 @@ def detect_drift(
                         dict(feature=feature, time_cutoff=time_cutoffs[ix])
                     )
         self.drift_alerts_count = len(self.drift_alerts)
+
+    def _check_missing_values_proportions(
+        self,
+        baseline: Union[list, np.ndarray, pd.Series],
+        new: Union[list, np.ndarray, pd.Series]
+    ):
+        """
+        This method gets the missing values frequencies from the original arrays and compares missing values frequencies
+        :param baseline: Baseline (old period / train set) vector
+        :param new: Target (future period / test set) vector
+        """
+        # Transform inputs into pandas series
+        baseline = pd.Series(baseline)
+        new = pd.Series(new)
+
+        # Calculate proportions of missing values
+        base_missing_prop = baseline.isna().mean()
+        new_missing_prop = new.isna().mean()
+
+        # Validate missing values proportions
+        if (base_missing_prop != 0 and new_missing_prop != 0 and
+            (new_missing_prop > base_missing_prop * (1 + self._significance) or
+             base_missing_prop > new_missing_prop * (1 + self._significance))) \
+           or (base_missing_prop == 0 and new_missing_prop > self._significance) \
+           or (new_missing_prop == 0 and base_missing_prop > self._significance):
+            print(f"Drift detected in missing values proportions")
```

Comment:
The message here is not indicative enough because the user won't know which feature is this related to. The messaging should be placed in the `detect_drift` function with `{feature}` as a variable.